script/1_0_2current_en_script/Common/00_00.txt

```diff
@@ -244,7 +244,7 @@ The woman crossed her arms and said confidently.
 "Well, it must be fate for us to meet like this, so why don't you keep me company for a bit? My name's Aozaki Aoko. You?"
 <Page65>
 The woman reached out to me casually as if we were friends.
-I couldn't find any reason to refuse, so I told her my name was Tohno Shiki and let her grasp suck me off the ground. // to be rechecked
+I couldn't find any reason to refuse, so I told her my name was Tohno Shiki and let her outstretched hand engulf mine.
```

Comment:
I think the idea here is that his hand is so much smaller than hers, not that she is yoinking him up, but could be wrong

> 　断る理由も見当たらなくて、僕は遠野志貴と名前をつげて、吸いこまれるように手のひらを握り返した。

Reply:
yeah this is right, mis-tl on my part